local/src/test/resources/kafka-1-creds/kafka-1_keystore_creds

```diff
@@ -0,0 +1 @@
+confluent
```

Comment:
why are we checking in secrets?

Reply:
In our ITs we have added tests that use SSL authentication and encryption. So these files are the resources which are needed for the tests to run. This is specific to the tests only. 
We are doing the same in one of the earlier commits(https://github.com/confluentinc/kafka-images/tree/master/kafka/test/fixtures/secrets)